### PR TITLE
Refactor: Streamline getAllIPs in InetUtils

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

File: source/agora/utils/InetUtils.d [hunk 1]
```diff
@@ -77,26 +90,7 @@ mixin template AddGetAllIPs()
...
-    mixin AddGetAllIPs;
-}
-version (linux)
-{
-    import core.sys.linux.ifaddrs;
-    import core.sys.posix.netdb;
-
-    mixin AddGetAllIPs;
-}
 version (Windows)
```
Comment (linked0) [inline source/agora/utils/InetUtils.d]: Indentation is needed.

Comment (Geod24) [reply source/agora/utils/InetUtils.d]: That code wasn't changed though, but I'll add a commit.

File: source/agora/utils/InetUtils.d [hunk 2]
```diff
@@ -41,11 +41,24 @@ public enum HostType : ubyte
...
+        version (OSX)
+        {
+            import core.sys.darwin.ifaddrs;
+            import core.sys.posix.netinet.in_;
+            import core.sys.posix.sys.socket;
+        }
+        version (linux)
+        {
+            import core.sys.linux.ifaddrs;
+            import core.sys.posix.netdb;
```
Comment (kchulj) [inline source/agora/utils/InetUtils.d]: Can this be removed due to the same import in L48?

Comment (Geod24) [reply source/agora/utils/InetUtils.d]: Added a commit.

**Review by linked0 — APPROVED** [review]
LGTM

**Review by kchulj — APPROVED** [review]
LGTM

**Pull request merged**